crates/ide-assists/src/handlers/unwrap_block.rs

```diff
@@ -734,13 +776,41 @@ fn main() {
 fn main() {
     let x = bar;
 }
+"#,
+        );
+        check_assist(
+            unwrap_block,
+            r#"
+fn main() -> i32 {
+    let _ = {$01; 2};
+}
+"#,
+            r#"
+fn main() -> i32 {
+    1;
+    let _ = 2;
+}
+"#,
+        );
+        check_assist(
+            unwrap_block,
+            r#"
+fn main() -> i32 {
+    let mut a = {$01; 2};
+}
+"#,
+            r#"
+fn main() -> i32 {
+    1;
+    let mut a = 2;
+}
 "#,
         );
     }
 
     #[test]
+    // https://github.com/rust-lang/rust-analyzer/issues/13679
```

Comment:
Please keep these inside the function to match how we do it in the rest of the code base